lib/Button/styles.scss

```diff
@@ -168,6 +168,22 @@ $color-btn-contained--bg: white !default;
   }
 }
 
+.button--purple-text {
```

Comment:
Should this not describe the colour (so the colour can be changed one day) and instead describe the purpose?

Reply:
Generally yes, but I don't think this one really serves a purpose (i.e. cancel/danger = red), its more I want a purple button because the rest of this is purple, I don't think purple signifies anything in particular in our UI. 

We also have a couple more that are named this way (light-grey, dark-grey)

Reply:
Yeah, our `Button` component is legacy and needs a heavy refactor soon 😄